google-api-client-apache-v5/pom.xml

```diff
@@ -29,8 +29,8 @@
                     <links>
                         <link>https://docs.oracle.com/javase/8/docs/api/</link>
                         <link>https://cloud.google.com/appengine/docs/standard/java/javadoc/</link>
-                        <link>https://googleapis.dev/java/google-http-client/${project.http.version}/</link>
-                        <link>https://googleapis.dev/java/google-oauth-client/${project.oauth.version}/</link>
+                        <link>https://javadoc.io/doc/com.google.http-client/google-http-client/${project.http.version}/</link>
+                        <link>https://javadoc.io/doc/com.google.oauth-client/google-oauth-client/${project.oauth.version}/</link>
```

Comment:
Isn't javadoc.io a third party (non-google) service? Do we want to rely on this? 

Do you know the impact if we just remove the two outdated `<link>` tags?

Reply:
I take back earlier [comment](https://github.com/googleapis/google-api-java-client/pull/2554#issuecomment-2610855549) . We are on [versions](https://github.com/googleapis/google-api-java-client/blob/0ac86743b2d6381b9314e36fed9377f0e890cd46/pom.xml#L538C1-L539C1) that exists from the c.g.c sites.  Updated the urls.